### PR TITLE
docs: Move the background image migration to its own section.

### PR DESCRIPTION
**Pull request by JeanMeche**

### CODE REVIEW

File: adev/src/content/guide/image-optimization.md [hunk 1]
```diff
@@ -17,6 +17,8 @@ In addition to optimizing the loading of the LCP image, `NgOptimizedImage` enfor
 * Warning if `width` or `height` have been set incorrectly
 * Warning if the image will be visually distorted when rendered
 
+In case you're using a background image in CSS, [start here](#how-to-migrate-your-background-image).
```
Comment (kara) [inline adev/src/content/guide/image-optimization.md:20]: Nit: sounds more natural

```suggestion
If you're using a background image in CSS, [start here](#how-to-migrate-your-background-image).
```

File: adev/src/content/guide/image-optimization.md [hunk 2]
```diff
@@ -385,15 +400,7 @@ Note that in the above example, we've invented the 'roundedCorners' property nam
...
-Here's a simple step-by-step process for migrating from `background-image` to `NgOptimizedImage`. For these steps, we'll refer to the element that has an image background as the "containing element":
-
-1) Remove the `background-image` style from the containing element.
-2) Ensure that the containing element has `position: "relative"`, `position: "fixed"`, or `position: "absolute"`.
-3) Create a new image element as a child of the containing element, using `ngSrc` to enable the `NgOptimizedImage` directive.
-4) Give that element the `fill` attribute. Do not include a `height` and `width`.
-5) If you believe this image might be your [LCP element](https://web.dev/lcp/), add the `priority` attribute to the image element.
-
-You can adjust how the background image fills the container as described in the [Using fill mode](#using-fill-mode) section.
+For a step-by-step process for migration from `background-image` to `NgOptimizedImage`, report to the [How to migrate your background image](#how-to-migrate-your-background-image) section above.
```
Comment (kara) [inline adev/src/content/guide/image-optimization.md:403]:
```suggestion
For a step-by-step process for migration from `background-image` to `NgOptimizedImage`, see the [How to migrate your background image](#how-to-migrate-your-background-image) section above. 
```

**Review by kara — APPROVED** [review]
LGTM overall, just some small nits. Thanks for picking this up!

**Review by AndrewKushnir — APPROVED** [review]

**Comment by dylhunn** [comment]
This PR was merged into the repository by commit c19a4e7a9687774b5e25b1b56e8ed46fb98d8ddb.

The changes were merged into the following branches: main, 18.0.x

**Pull request closed without merging**